app/src/main/java/com/helio/app/EditBlindsActivity.java

```diff
@@ -0,0 +1,59 @@
+package com.helio.app;
```

Comment:
This file isn't being used for anything. It also should be a fragment instead of an activity.